csrc/scheduler/multi_matmul.cpp

```diff
@@ -1006,16 +1006,179 @@ class MultipleMatmulScheduler {
     scheduleBranch(bs_, bcw_smems_, params_->supported_vec_size.b);
   }
 
+  // Schedule acr and bcr
+  void scheduleLdMatrixLoads() {
```

Comment:
Is this used?

Reply:
Ah you're right. It is no longer used. I'll remove it.